### PR TITLE
[MPDX-8094] Add setup finish page

### PR DESCRIPTION
**Pull request by canac**
## Description

* Add setup finish page
* The logic on the tools page to notice the `?setup=1` query parameter will be done as part of MPDX-8095
* Adding logic to redirect to this page can either be done in #1002 or in a future PR

[MPDX-8094](https://jira.cru.org/browse/MPDX-8094)

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [x] I have requested a review from another person on the project

### CODE REVIEW

File: pages/setup/finish.page.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    });
+
+  useEffect(() => {
+    setSetupPosition('finish');
+  }, []);
+
+  const handleNext = async () => {
+    await setSetupPosition('');
+    push(`/accountLists/${defaultAccountListId}/settings/integrations?setup=1`);
+  };
```
Comment (dr-bizz) [inline pages/setup/finish.page.tsx:43]: This should go to tools and not settings correct?

**Review by dr-bizz — COMMENTED** [review]
This looks great. I had one question as it looks like you are redirect to settings instead of tools.

File: pages/setup/finish.page.tsx (hunk 1 above)
Comment (canac) [reply pages/setup/finish.page.tsx]: Good catch, thank you!

**Review by dr-bizz — APPROVED** [review]
Looks great

**Pull request merged**